packages/playground/website/src/main.tsx

```diff
@@ -111,280 +63,25 @@ if (currentConfiguration.wp === '6.3') {
-								<MenuGroup>
-									<ResetSiteMenuItem
-										storage={currentConfiguration.storage}
-										onClose={onClose}
-									/>
-									<ReportError
-										onClose={onClose}
-										disabled={offline}
-									/>
-									<DownloadAsZipMenuItem onClose={onClose} />
-									<RestoreFromZipMenuItem onClose={onClose} />
-									<GithubImportMenuItem
-										onClose={onClose}
-										disabled={offline}
-									/>
-									<GithubExportMenuItem
-										onClose={onClose}
-										disabled={offline}
-									/>
-									<ViewLogs onClose={onClose} />
-									<MenuItem
-										icon={external}
-										iconPosition="left"
-										aria-label="Go to Blueprints Builder"
-										href={
-											[
-												joinPaths(
-													document.location.pathname,
-													'builder/builder.html'
-												),
-												'#',
-												btoa(
-													JSON.stringify(
-														blueprint
-													) as string
-												) as string,
-											].join('') as any
-										}
-										target="_blank"
-										disabled={offline}
-									>
-										Edit the Blueprint
-									</MenuItem>
-								</MenuGroup>
-								<MenuGroup label="More resources">
-									<MenuItem
-										icon={external}
-										iconPosition="left"
-										aria-label="Go to WordPress PR previewer"
-										href={
-											joinPaths(
-												document.location.pathname,
-												'wordpress.html'
-											) as any
-										}
-										target="_blank"
-										disabled={offline}
-									>
-										Preview WordPress Pull Request
-									</MenuItem>
-									<MenuItem
-										icon={external}
-										iconPosition="left"
-										aria-label="Go to a list of Playground demos"
-										href={
-											joinPaths(
-												document.location.pathname,
-												'demos/index.html'
-											) as any
-										}
-										target="_blank"
-										disabled={offline}
-									>
-										More demos
-									</MenuItem>
-									<MenuItem
-										icon={external}
-										iconPosition="left"
-										aria-label="Go to Playground documentation"
-										href={
-											'https://wordpress.github.io/wordpress-playground/' as any
-										}
-										target="_blank"
-										disabled={offline}
-									>
-										Documentation
-									</MenuItem>
-									<MenuItem
-										icon={external}
-										iconPosition="left"
-										aria-label="Go to the Playground git repository"
-										href={
-											'https://github.com/WordPress/wordpress-playground' as any
-										}
-										target="_blank"
-										disabled={offline}
-									>
-										GitHub
-									</MenuItem>
-								</MenuGroup>
-							</>
-						)}
-					</DropdownMenu>,
-				]}
-			/>
-		</PlaygroundContext.Provider>
+	const showSiteManager = query.get('view') === 'manager';
+	return showSiteManager ? (
```

Comment:
We can always render `<SiteManager>` here, it would just internally observe the current route and either show or hide the sidebar. This way React won't trash the Playground `<iframe>` when the state changes and also we'll be able to use css transitions.

Reply:
I replaced this with the Navigator component.